modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/EditorConfigContributorCETImplFactoryImpl.java

```diff
@@ -50,11 +52,19 @@ public UnicodeProperties getUnicodeProperties(
 			true
 		).put(
 			"editorConfigKeys",
-			ParamUtil.getString(portletRequest, "editorConfigKeys")
+			StringUtil.merge(
+				ParamUtil.getStringValues(portletRequest, "editorConfigKeys"),
+				StringPool.NEW_LINE)
 		).put(
-			"editorNames", ParamUtil.getString(portletRequest, "editorNames")
+			"editorNames",
+			StringUtil.merge(
+				ParamUtil.getStringValues(portletRequest, "editorNames"),
+				StringPool.NEW_LINE)
 		).put(
-			"portletNames", ParamUtil.getString(portletRequest, "portletNames")
+			"portletNames",
+			StringUtil.merge(
+				ParamUtil.getStringValues(portletRequest, "portletNames"),
+				StringPool.NEW_LINE)
```

Comment:
Nice catch :tada:, we lost those bits at some point from the PoC 

@juanjofgliferay @boton this is the thing we were looking for this morning. No more actions on your side